splight_lib/models/line.py

```diff
@@ -0,0 +1,49 @@
+from splight_lib.models import (
+    AssetParams,
+    AssetRelationship,
+    Attribute,
+    Metadata,
+)
+from splight_lib.models.database_base import SplightDatabaseBaseModel
+
+
+class Line(AssetParams, SplightDatabaseBaseModel):
+    active_power: None | Attribute = None
```

Comment:
Sorry for this change but I would do it in the other way
```python
class Line(AssetParams, SplightDatabaseBaseModel):
    active_power: Attribute | None = None
    ...
```
Based on my understanding this is the correct way to use the optional type hints. Also the order is relevant to pydantic to run the validations